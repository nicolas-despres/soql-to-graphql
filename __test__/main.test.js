var converter = require ('../build/src/main.js').default;

describe('converter function', () => {
  

  // Assert greeter result
  it('Sample conversion test', () => {
  
    expect(converter('select Id, Name, (select Id from Opportunities) from Account limit 3')).toMatchSnapshot();
  });
});