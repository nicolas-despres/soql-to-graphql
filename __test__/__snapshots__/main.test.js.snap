// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`greeter function Sample conversion test 1`] = `
"query {
    uiapi {
        query {
            Account (first: 3) {
                edges {
                    node {
                        Id
                        Name {
                            value
                        }
                        Opportunities {
                            edges {
                                node {
                                    Id
                                }
                            }
                        }
                    }
                }
            }
        }
    }
}"
`;
